cmd/kops/util/factory.go

```diff
@@ -274,9 +277,9 @@ func (f *Factory) buildRESTConfig(ctx context.Context, cluster *kops.Cluster) (*
 		return nil, err
 	}
 
-	// Generate a relatively short-lived certificate / kubeconfig
-	createKubecfgOptions := kubeconfig.CreateKubecfgOptions{
-		Admin: 1 * time.Hour,
+	// keep backwards compatibility with the old default
```

Comment:
We could just raise the value to 24 hours or 72 hours.  This credential isn't written to disk IIRC so it's pretty secure.

We could also auto-rotate it near expiry, but that's more complicated that just bumping the value.

Reply:
Is --admin flag even needed after 24hours?

Reply:
I decided to keep it configurable, but default value for 18 hours. That should be enough to 99% usecases and that 1% can use flag to make it higher